README.md

```diff
@@ -93,8 +93,28 @@ Note: All example values are the actual values for Ethereum Mainnet and the stag
 
 Run unit tests on the local Hardhat network:
 
+To test DssVestTopUp.sol:
+
+```bash
+npm run test:topup
+```
+
+To test DssCronKeeper.sol:
+
+```bash
+npm run test:keeper
+```
+
+For end-to-end testing:
+
+```bash
+npm run test:e2e
+```
+
+To execute the E2E test, you need a localhost network while running a mainnet fork in a separate terminal:
+
 ```bash
-npm run test
+RPC_URL=https://example-rpc.com yarn net:fork
```

Comment:
`npm run test:e2e` should be enough. Why would you need to run this command?

Reply:
How else can you run tests on a forked chain? 

`
npx hardhat test test/E2E.test.ts --fork ${RPC_URL}
 --fork-block-number 8273687
`

This is giving me  `Error HH305: Unrecognized param --fork`. So I thought that maybe we should run a node that has a forked chain.  And we cant just fork the hardhat network because this way we are gonna have run the unit tests on the forked chain too.